plugins/mediafoundation/capture/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.28...3.30)
+
+find_package(FFmpeg REQUIRED avcodec avutil)
+find_package(WIL REQUIRED)
+
+add_library(win-mf MODULE)
+add_library(OBS::mf ALIAS win-mf)
+
+target_sources(win-mf PUBLIC mf-plugin.cpp win-mf.cpp)
```

Comment:
This will need to be renamed as it will clobber the plugin name used by the MediaFoundation encoder plugin in development as well.

Will reply back once I have guidance for what the correct plugin name/ID and subdirectory should be to get a proper, final, name.